project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.8
```

Comment:
We were just on a very old version, so figured I would update it while mucking around the build settings.